### PR TITLE
update the translation of `Array.tospliced()`

### PR DESCRIPTION
**Pull request by robert-jx**
### Description
update the translation of array.tospliced()

update the translation of array.tospliced()
### Additional details
none

### CODE REVIEW

**Comment by robert-jx** [comment]
@JasonLamv-t

File: files/zh-cn/web/javascript/reference/global_objects/array/tospliced/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    如果你没有指定任何元素，`toSpliced()` 只会从数组中删除元素。
+
+### 返回值
+
+一个新数组，包含 `start`，`item1`，`item2`，…，`itemN` 之前的所有元素，以及 `start + deleteCount` 之后的所有元素。
+
+## 描述
+
+`toSpliced()` 方法，像 `splice()` 一样，一次做多件事：它从数组中移除给定数量的元素，从给定的索引开始，然后在相同的索引中插入给定的元素。但是，它返回一个新数组，而不是修改原始数组。因此，此方法不会返回已删除的元素。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/tospliced/index.md:47]:
```suggestion
`toSpliced()` 方法与 `splice()` 类似，可以同时完成多个操作：在数组中给定的索引开始移除指定数量的元素，然后在相同的索引处插入给定的元素。但是，它返回一个新数组，而不是修改原始数组。因此，此方法不会返回已删除的元素。
```

File: files/zh-cn/web/javascript/reference/global_objects/array/tospliced/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+`toSpliced()` 方法不会产生[稀疏数组](/zh-CN/docs/Web/JavaScript/Guide/Indexed_collections#稀疏数组)。如果原始数组是稀疏的，在新数组中空槽将会被替换成 `undefined`。
+
+`toSpliced()` 方法是[通用的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#通用数组方法)。它只期望 `this` 值具有 `length` 属性和整数键属性。
+
+## 示例
+
+### 删除，添加和替换元素
+
+你可以通过使用 `toSpliced()` 来删除，添加和替换数组中的元素，并生成一个比使用 `slice()` 和 `concat()` 更有效的数组。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/tospliced/index.md:57]:
```suggestion
你可以通过使用 `toSpliced()` 来删除、添加和替换数组中的元素，并生成一个的数组，这比使用 `slice()` 和 `concat()` 更高效。
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions

File: files/zh-cn/web/javascript/reference/global_objects/array/tospliced/index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+`toSpliced()` 方法与 `splice()` 类似，可以同时完成多个操作：在数组中给定的索引开始移除指定数量的元素，然后在相同的索引处插入给定的元素。但是，它返回一个新数组，而不是修改原始数组。因此，此方法不会返回已删除的元素。
+
+`toSpliced()` 方法不会产生[稀疏数组](/zh-CN/docs/Web/JavaScript/Guide/Indexed_collections#稀疏数组)。如果原始数组是稀疏的，在新数组中空槽将会被替换成 `undefined`。
+
+`toSpliced()` 方法是[通用的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#通用数组方法)。它只期望 `this` 值具有 `length` 属性和整数键属性。
+
+## 示例
+
+### 删除，添加和替换元素
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/array/tospliced/index.md:55]:
```suggestion
### 删除、添加和替换元素
```

File: files/zh-cn/web/javascript/reference/global_objects/array/tospliced/index.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+你可以通过使用 `toSpliced()` 来删除、添加和替换数组中的元素，并生成一个的数组，这比使用 `slice()` 和 `concat()` 更高效。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/array/tospliced/index.md:57]:
```suggestion
你可以通过使用 `toSpliced()` 来删除、添加和替换数组中的元素，并生成一个新的数组，这比使用 `slice()` 和 `concat()` 更高效。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**